extension.js

```diff
@@ -149,14 +142,39 @@ function getTranslationsPromiseArrayLine(
 }
 
 /**
- * Returns user settings Preferred language
- *
- * @returns {string}
+ * Returns user settings Preferred language.
+ * If user hasn't set preferred lang. Prompt to set.
  */
-function getPreferredLanguage() {
-  return vscode.workspace
+async function getPreferredLanguage() {
+  var pref = vscode.workspace
     .getConfiguration("vscodeGoogleTranslate")
     .get("preferredLanguage");
+
+  if (!pref) {
+    return setPreferredLanguage()
+  }
+  return pref;
```

Comment:
I think it could be like this:
```js
return vscode.workspace
      .getConfiguration("vscodeGoogleTranslate")		      
      .get("preferredLanguage");
   || setPreferredLanguage();
```

Reply:
beautiful!